vue-training/src/views/index/index.js

```diff
@@ -2,12 +2,98 @@ import Vue from 'vue'
 
 import { installServiceWorker } from '../../serviceWorkerInstaller'
 
+const TechBox = () => import('../../components/TechBox')
+const TechTitle = () => import('../../components/TechTitle')
+
 import './index.pug'
 import './index.scss'
 
-// eslint-disable-next-line
+// eslint-disable-next-line no-unused-vars
 const vm = new Vue({
-  el: '#app'
+  el: '#app',
+  components: {
+    TechBox,
+    TechTitle
+  },
+  data: {
+    techs: [
+      {
+        id: 1,
```

Comment:
you can delete this and use the index inside the v-for (think that here frontend-Tools and reactjs have the same id 😉